### PR TITLE
Fixes and enhancments

### PR DESCRIPTION
**Pull request by k0da**
* updates Wrangler to 0.7.2
* Fixes crednetials for git clone
* update to go 1.15
* Watch for configmap and secret and reconcile state if related are changed

### CODE REVIEW

**Comment by luthermonson** [comment]
can you address the build fail @k0da https://drone-pr.rancher.io/rancher/terraform-controller/84/1/2

**Comment by k0da** [comment]
@luthermonson sure, looking at them

File: Dockerfile.dapper [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.1-alpine3.9
+FROM golang:1.14.8-alpine
```
Comment (luthermonson) [inline Dockerfile.dapper:1]: make go versions consistent, let's just go to 1.15, change in go mod file too

Comment (k0da) [reply Dockerfile.dapper]: @luthermonson done

**Review by luthermonson — APPROVED** [review]

**Pull request merged**